modules/data.remote/R/merge_image_tiles.R

```diff
@@ -8,7 +8,9 @@
 #' @param out.path  character: physical path to the folder that contains converted and merged images.
 #' @param band.name character: band name of the image. Default is NULL.
 #' @param just.band.name boolean: if we just want the band names of the image file. Default is TRUE.
-#' @param keep.files Boolean: if we want to keep the image tiles at the end.
+#' @param keep.files boolean: if we want to keep the image tiles at the end.
+#' @param ignore.conversion boolean: if we want to ignore the image conversion.
```

Comment:
Tiny nit: in R the standard nomenclature for TRUE/FALSE variables is "logical" rather than "boolean".

Less of a nit, but still small: It's not obvious to me from this description what "ignore" means in this context. am I interpreting right that it leads to not performing a call to `gdal_conversion`, and if so would "skip conversion" be a clearer way to phrase it?